### PR TITLE
Add ability to disable colors

### PR DESCRIPTION
**Pull request by charlierudolph**
@jbpros

### CODE REVIEW

File: lib/cucumber/util/colors.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
-var colors = require('colors/safe');
+function Colors (useColors) {
```
Comment (samccone) [inline lib/cucumber/util/colors.js]: and now I must raise the question, what does this level of indirection give us :)

Comment (charlierudolph) [inline lib/cucumber/util/colors.js]: Okay. I guess we could just put the following somewhere and set `colors.enabled` in our argument parser. Where do you think the following code should go? 
```js
colors.setTheme({
  'comment'   : 'grey',
  'failed'    : 'red',
  'passed'    : 'green',
  'pending'   : 'yellow',
  'skipped'   : 'cyan',
  'tag'       : 'cyan',
  'undefined' : 'yellow'
});
```

File: lib/cucumber/cli.js [hunk 2]
```diff
@@ -89,6 +89,8 @@ function Cli(argv) {
 \n\
 --no-source                   Don't print the source uris.\n\
 \n\
+--[no-]colors                 Enable/disable colors in output.\n\
```
Comment (jbpros) [inline lib/cucumber/cli.js]: That's `--no-color` in Cucumber ruby. Let's stay consistent (even though `--no-colors` is better, imo). Or both would be fine as well.

File: lib/cucumber/util/colors.js (hunk 1 above)
Comment (jbpros) [inline lib/cucumber/util/colors.js]: I think it's nice to keep all the color settings together like this.

File: lib/cucumber/util/colors.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+  var colors = require('colors/safe');
```
Comment (jbpros) [inline lib/cucumber/util/colors.js]: Any reason to keep this within the function and not at the top of the file in a more conventional way?

File: lib/cucumber/cli.js (hunk 2 above)
Comment (charlierudolph) [inline lib/cucumber/cli.js]: I don't think we need to copy cucumber-ruby to a T. We already specify formats differently. I also think no-colors is better, so why not as cucumber-ruby to rename? cucumber-jvm uses `--monochrome` as well. 

I like the idea of keeping these implementations interfaces the same, but these are also built at different times and I'd prefer not to be locked to the cucumber-ruby interface.

File: lib/cucumber/util/colors.js (hunk 3 above)
Comment (charlierudolph) [inline lib/cucumber/util/colors.js]: no. I'm fine moving it up

File: lib/cucumber/cli.js (hunk 2 above)
Comment (jbpros) [inline lib/cucumber/cli.js]: Well okay, if JVM uses this different `--monochrome` flag (I don't think "monochrome" is the right term for this), let's use our preferred `--no-colors`  then.

**Comment by jbpros** [comment]
:+1:

**Pull request closed without merging**